bundles/org.openhab.binding.yamahamusiccast/README.md

```diff
@@ -0,0 +1,162 @@
+
+N/A
+
+## Thing Configuration
+
+| Parameter              | type    | description                                             | Advanced | Required      |
+|------------------------|---------|---------------------------------------------------------|----------|---------------|
+| configHost             | String  | IP address of the Yamaha model (AVR, ...)               | false    | true          |
+| configSyncVolume       | Boolean | Sync volume across linked models (default=false)        | true     | false         |
+
+
+## Channels
+
+| channel               | type   | description                                                        |
+|-----------------------|--------|--------------------------------------------------------------------|
+| channelPower          | Switch | Power ON/OFF                                                       |
+| channelMute           | Switch | Mute ON/OFF                                                        |
+| channelVolume         | Dimmer | Volume as % (recalculated based on Max Volume Model)               |
+| channelVolumeAbs      | Number | Volume as absolute value                                           |
+| channelInput          | String | See below for list                                                 |
+| channelSoundProgram   | String | See below for list                                                 |
+| channelSelectPreset   | String | Select Netradio/USB preset (fetched from Model)                    |
+| channelSleep          | Number | Fixed values for Sleep : 0/30/60/90/120                            |
+| channelMCServer       | String | Select your MusicCast Server or set to Standalone                  |
+| channelUnlinkMCServer | Switch | Disband MusicCast Link on Master                                   |
+| channelRecallScene    | Number | Select a scene (create your own dropdown list!)                    |
+| channelPlayer         | Player | PLAY/PAUSE/NEXT/PREVIOUS/REWIND/FASTFORWARD                        |
+| channelArtist         | String | Artist                                                             |
+| channelTrack          | String | Track                                                              |
+| channelAlbum          | String | Album                                                              |
+| channelAlbumArt       | Image  | Album Art                                                          |
+| channelRepeat         | String | Toggle Repeat. Available values: Off, One, All                     |
+| channelShuffle        | String | Toggle Shuffle. Availabel values: Off, On, Songs, Album            |
+
+
+| Zones                | description                                          |
+|----------------------|------------------------------------------------------|
+| Zone1-4              | Zone 1 to 4 to control Power, Volume, ...            |
+| playerControls       | Separate zone for Play, Pause, ...                   |
+
+## Full Example
```

Comment:
I think it's more reasonable to put the input list here as you referenced to it above. The Full Example chapter is good at the end.

Reply:
Next push will contain this requested change.

Reply:
Done, push has been done.